datamap/mv/core/src/main/scala/org/apache/carbondata/mv/datamap/MVHelper.scala

```diff
@@ -118,6 +122,43 @@ object MVHelper {
     DataMapStoreManager.getInstance().saveDataMapSchema(dataMapSchema)
   }
 
+  private def validateMVQuery(sparkSession: SparkSession,
+      logicalPlan: LogicalPlan) {
+    val dataMapProvider = DataMapManager.get().getDataMapProvider(null,
+      new DataMapSchema("", DataMapClassProvider.MV.getShortName), sparkSession)
+    dataMapProvider
+    var catalog = DataMapStoreManager.getInstance().getDataMapCatalog(dataMapProvider,
+      DataMapClassProvider.MV.getShortName).asInstanceOf[SummaryDatasetCatalog]
```

Comment:
Does casting `null` to `SummaryDatasetCatalog` ok?

Reply:
ok